plumgrid_playbooks/roles/plumgrid_common/tasks/main.yml

```diff
@@ -45,51 +40,37 @@
     file: /tmp/GPG-KEY
     state: present
 
-# Update repositories
-- name: Running apt-update
```

Comment:
@JaveriaK it's constant across different version of ansible. Even if you want to do it at the start of the playbook, you can do in Ansible way, which is:

```yaml
- name: Update the cache
   apt:
       update_cache: yes
```